pulpcore/app/models/upload.py

```diff
@@ -1,24 +1,28 @@
 import hashlib
 import os
 
+from django.conf import settings
 from django.core.files.base import ContentFile
+from django.core.files.storage import FileSystemStorage
 from django.db import models
 from rest_framework import serializers
 
 from pulpcore.app.models import BaseModel
 
+tmp_fs = FileSystemStorage(location=settings.CHUNKED_UPLOAD_DIR)
```

Comment:
What's the reasoning behind defining this variable?

Reply:
I wanted to clearly state that this FileSystemStorage is temporary. A may do the following in the model as well:
```suggestion
    file = models.FileField(null=False, max_length=255, storage=tmp_fs = FileSystemStorage(location=settings.CHUNKED_UPLOAD_DIR))
```